__tests__/hooks/useAuth.test.tsx

```
@@ -62,6 +112,10 @@ describe('useAuth Hook', () => {
 
   afterEach(() => {
     jest.restoreAllMocks();
+    // Restore original location if changed
+    if (originalLocation && window.location !== originalLocation) {
+      (window as any).location = originalLocation;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)